views/partials/_tracking-head.njk

```diff
@@ -1,4 +1,6 @@
 {% if GTM_TAG %}
+<link rel="preconnect" href="https://www.google-analytics.com" />
```

Comment:
We don't _generally_ close `link` tags (we're not 100% consistent on this, but the vast majority aren't)